static/app/components/events/interfaces/crashContent/exception/content.tsx

```diff
@@ -38,28 +42,27 @@ type Props = {
   threadId?: number;
 } & Pick<React.ComponentProps<typeof StackTrace>, 'groupingCurrentLevel'>;
 
-type CollapsedExceptionMap = Record<number, boolean>;
+type ExceptionRenderStateMap = Record<number, boolean>;
 
-const useCollapsedExceptions = (values?: ExceptionValue[]) => {
-  const [collapsedExceptions, setCollapsedSections] = useState<CollapsedExceptionMap>(
+const useHiddenExceptions = (values?: ExceptionValue[]) => {
```

Comment:
i changed the names from collapsed to hidden since i thought it might be a lilttle confusing that this hook is used as part of the functionality for showing/hiding child exceptions, while a user may also additionally collapse/open the each individual exception fold section